### ISSUE TITLE
Choose specific library

### ISSUE THREAD

**Issue by Shayne55434** [maykar/plex_assistant#83]
I looked through your code and it appears that you get a list of all available libraries within Plex. However, I can't see if there's a way to choose which library to play media from. I've tried a few commands with no luck. 

For example, I have "4k Movies" and "Movies" libraries and I may want to play a 4k version instead of the defaulted "Movies" version. Is this possible?

Thanks for all your work! I love it!

**Comment by Shayne55434** [comment]
Any thoughts on implementing this?

**Comment by maykar** [comment]
Yes, this is on my list and shouldn't be that difficult to implement. Would just require an extra phrase like"...from the 4k library" etc.
